style.css

```diff
@@ -353,6 +348,137 @@ header {
   box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
 }
 
+.popup-container {
+  position: fixed;
+  transform: translate(0%, 0%) scale(0);
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 21;
+  background-color: rgba(64, 83, 252, 0.24);
+  backdrop-filter: blur(4px);
+  transition: 0.3s ease-in-out;
+}
+
```

Comment:
- [ ] Kindly make sure to add width on right and left sides of the image. As you can see the image is only having a height at the top and bottom and according to the Figma design, it should be in the centre.
- [ ] Kindly make the corners of the image container in the popup window curve at the end as per the Figma design.
- Figma design
<img width="303" alt="Screenshot 2022-04-06 at 2 02 21 AM" src="https://user-images.githubusercontent.com/70269507/161866481-11360c65-fb85-45ee-9d17-772212518eae.png">
Your design:
<img width="558" alt="Screenshot 2022-04-06 at 2 04 53 AM" src="https://user-images.githubusercontent.com/70269507/161866514-4380aabc-34de-493a-8795-cc48c6803e4b.png">

Reply:
Hi @sja-thedude,
I appreciate the feedback you provide on this project and I want to ask the following things:

- Are you suggesting that every content of the work section has to be dynamic?
- I don't understand very well what you're trying to mean by "make the corners of the image container in the popup window curve at the end as per the Figma design"

Reply:
Hi @RWUBAKWANAYO, 
Yes I am suggesting you to make the entire work section dynamic.

I try to mean that as you can see in the images below that the figma design have border radius which makes it rounded corners.

Your design:
![image](https://user-images.githubusercontent.com/70269507/161871979-b0cb29a6-d8bc-4ae3-8cd9-3f55bdbee29f.jpeg)

Figma design:
![image](https://user-images.githubusercontent.com/70269507/161871997-e4593cfc-6b3f-4c88-b25f-f3f29302a09d.jpeg)